frontend/public/locales/en/translation.json

```diff
@@ -1062,6 +1062,7 @@
     "deleting": "Deleting",
     "Deleting": "Deleting",
     "Deleting <italic>{{policyAutomation}}</italic> is irreversible. Any associated Ansible job will be deleted.": "Deleting <italic>{{policyAutomation}}</italic> is irreversible. Any associated Ansible job will be deleted.",
+    "Deleting this policy may remove some related objects on the managed cluster(s).": "Deleting this policy may remove some related objects on the managed cluster(s).",
```

Comment:
Deleting this policy might delete some related objects on the managed cluster(s).